### PR TITLE
Disabled built-in HTML validation of forms

### PR DESCRIPTION
**Pull request by ognyan-kostadinov**
**Component**:

**Description**:

**Design**:

**Breaking Changes**:

- [] Breaking Changes

---

Closes: #ISSUE_NUMBER

### CODE REVIEW

File: src/lib/components/form/Form.component.tsx [hunk 1]
```diff
@@ -264,7 +264,7 @@ const PageForm = forwardRef<HTMLFormElement, PageFormProps>(
     const requireMode = useContext(RequireModeContext);
     return (
       <ScrollbarWrapper>
-        <PageFormWrapper {...formProps} ref={ref}>
+        <PageFormWrapper {...formProps} noValidate ref={ref}>
```
Comment (JBWatenbergScality) [inline src/lib/components/form/Form.component.tsx:270]: It should probably also be set on the TabForm. Also as we enforce it at this level, it should not be allowed by the Form props type (aka `Omit<FormProps, 'novalidate'>` is necessary)

Comment (ognyan-kostadinov) [reply src/lib/components/form/Form.component.tsx:270]: I'm confused, should I apply this omission to PageForm or PageFormWrapper?

Comment (JBWatenbergScality) [reply src/lib/components/form/Form.component.tsx:270]: From PageForm props

File: src/lib/components/form/Form.component.tsx [hunk 2]
```diff
@@ -22,7 +22,10 @@ import { Text } from '../text/Text.component';
 const DESCRIPTION_PREFIX = 'describe-';
 const maxWidthTooltip = { maxWidth: '20rem' };
 
-type FormProps = Omit<HTMLProps<HTMLFormElement>, 'ref' | 'as'> & {
+type FormProps = Omit<
+  HTMLProps<HTMLFormElement>,
+  'ref' | 'as' | 'noValidate' | 'formNoValidate'
```
Comment (JBWatenbergScality) [inline src/lib/components/form/Form.component.tsx:27]: In practice `formNoValidate` is an input property not a form one. But ok for me this is really minor !

**Review by JBWatenbergScality — APPROVED** [review]

File: src/lib/components/form/Form.component.tsx (hunk 2 above)
Comment (ognyan-kostadinov) [reply src/lib/components/form/Form.component.tsx:27]: Yes, I saw that, but the IDE was suggesting it on the Form component for some reason, so I've made the decision to remove it in order to not mislead people in the future

**Pull request merged**